RAGchain/benchmark/dataset/search_qa.py

```diff
@@ -100,8 +108,8 @@ def ingest(self, retrievals: List[BaseRetrieval], db: BaseDB, ingest_size: Optio
 
     def evaluate(self, **kwargs) -> EvaluateResult:
         question = self.qa_data['question'].tolist()
-        retrieval_gt = [[uuid.UUID(key) for key in key_lst] for key_lst in self.qa_data['key']]
-        answer_gt = [[answer[0] for answer in answer_lst] for answer_lst in self.qa_data['answers']]
+        retrieval_gt = [[uuid.UUID(gt) for gt in gt_lst] for gt_lst in self.qa_data['retrieval_gt']]
```

Comment:
self.qa_data.apply(lambda row: list(map(lambda x: uuid.UUID(x), row['retrieval_gt'])), axis=1)

Maybe work? I didn't check;

Reply:
It work!
![image](https://github.com/Marker-Inc-Korea/RAGchain/assets/60510718/e66c1cdb-6ffe-43dd-9145-9327b18282c3)